lang/squeak/Portfile

```diff
@@ -83,13 +83,12 @@ post-destroot {
     
     # fix bad doc install path (should use --docdir or --datarootdir during configure)
     xinstall -d ${destroot}${datadir}
-    move ${destroot}${prefix}/doc ${destroot}${prefix}/share
+    move ${destroot}${prefix}/doc/${name} ${destroot}${datadir}/doc
```

Comment:
You are changing where the port installs documentation. It used to be installed into /opt/local/share/doc/squeak. (You can see this in squeak-3.9_1.darwin_10.x86_64.tbz2 available at http://packages.macports.org/squeak/) After your change, it will be installed into /opt/local/share/squeak/doc. If this change is desirable (why would it be?) then the `revision` would need to be increased.

Reply:
No preference here, but existing code just does not work. I can try changing it in a way to install into a supposed location of the earlier version.

Reply:
Changed to share/doc/squeak now.